### DISCUSSION TITLE
How to add Prolarr in search providers?

### DISCUSSION THREAD

**Discussion by AlcyonTaf** [pymedusa/Medusa#10199] [Q&A]
Hello,
I tried to add Prowlarr as a search provider for torrent but was unsuccessful.
No matter how much I looked, but I can't find how to do it.
Can someone explain to me?
thank you

**Comment by p0psicles** [comment]
Prowlarr is not a provider. Prowlarr is an application in where your can configure providers. You need to install the application, for ex. in docker. And then in medusa you connect to prowlarr. Then you'll get a list of "prowlarr" providers from which you can enable them.

**Reply by p0psicles** [reply]
https://github.com/Prowlarr/Prowlarr

**Reply by AlcyonTaf** [reply]
Sorry I did not have to be clear in the description of my problem.
I have well prowlarr to install and functional with docker. But I could not do the addition in Medusa in "configure Custom Prowlarr Providers".
I solved my problem, I was wrong in "prowler URL" : http://IPROWLAR:9696/1/api instead of http://IPROWLAR:9696/.

But now I have a problem with the categories in "Configure Torznab Providers". If I add a category (5070 for TV \ Anime) it is not used during the search.
Log in Prowlarr : /api?apikey=(removed)&t=search&limit=100&offset=0&cat=5010%2C5030%2C5040%2C7000&q=Demon+Slayer+S03E02: 200.OK (0 ms).

I will see if I can find the solution to this problem.
Thank you for your answers!

**Reply by gunther-neitzen** [reply]
was this ever solved?
